specification/communication/data-plane/Chat/preview/2024-03-15-preview/communicationserviceschat.json

```diff
@@ -0,0 +1,2068 @@
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Get chat thread": {
+            "$ref": "./examples/Threads_GetChatThread.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Deletes a thread.",
+        "operationId": "Chat_DeleteChatThread",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "Id of the thread to be deleted.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Request successful."
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Delete chat thread": {
+            "$ref": "./examples/Threads_DeleteChatThread.json"
+          }
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}/typing": {
+      "post": {
+        "tags": [
+          "TypingNotifications"
+        ],
+        "summary": "Posts a typing event to a thread, on behalf of a user.",
+        "operationId": "ChatThread_SendTypingNotification",
+        "consumes": [
+          "application/json",
+          "application/merge-patch+json"
```

Comment:
A post operation should not consume 'application/merge-patch+json' content type.

Reply:
good catch, i remember that was looking into why its there and someone said it can be removed.